### DISCUSSION TITLE
Introduce Gherkin Syntaxed

### DISCUSSION THREAD

**Discussion by pedro-pmda** [mammadataei/cypress-vite#46] [Ideas]
I would like to be able to run my e2e the cypress in vite tests using BDD. Now I do it through [the cucumber preprocessor ](https://www.npmjs.com/package/cypress-cucumber-preprocessor) using ,webpack but it would be helpful if this vite preprocessor allowed both.
Regards 🙂

**Comment by mammadataei** [comment]
Since cypress-vite uses Vite under the hood, we need a Vite plugin that compiles the cucumber `.feature` files. Similar to https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/esbuild.ts.
